lib/rules/no-multiple-empty-lines.js

```diff
@@ -34,6 +46,12 @@ module.exports = function(context) {
                 trimmedLines = lines.map(function(str) {
                     return str.trim();
                 });
+
+            // add the notEmpty lines in there with a placeholder
+            notEmpty.forEach(function(x, i) {
```

Comment:
I would just see if adding a line into the map function above to get node for current location and check if it's not TemplateLiteral would work. If it does, it feels like an easier solution.